### PR TITLE
roachtest: bump PredecessorVersion(20.2) to 20.1.5

### PR DESCRIPTION
**Pull request by arulajmani**
Release justification: change part of 20.1.5 release process
Release note: None

### CODE REVIEW

File: pkg/cmd/roachtest/test_runner.go [hunk 1]
```diff
@@ -1158,7 +1158,7 @@ func PredecessorVersion(buildVersion version.Version) (string, error) {
 	// (see runVersionUpgrade). The same is true for adding a new key to this
```
Comment (jlinder) [inline pkg/cmd/roachtest/test_runner.go:1158]: Does this comment need to be followed to update the fixtures (as was done [for these releases](https://github.com/cockroachdb/cockroach/pull/50770))?

Comment (arulajmani) [reply pkg/cmd/roachtest/test_runner.go:1158]: Yeah, I think so. Seems like checkpointing the 20.1.5 fixtures requires the binaries to be downloadable from the homepage -- so I'm gonna wait on this PR until that's done.

**Comment by arulajmani** [comment]
@jlinder sorry for the delay on closing the loop on this, I've updated the checkpointing error message as well so that we don't run into this problem again.

**Review by jlinder — APPROVED** [review]
LGTM

**Comment by arulajmani** [comment]
bors r=jlinder

**Comment by arulajmani** [comment]
closing this in favour of #54761

**Comment by arulajmani** [comment]
closing this in favour of #54761

**Pull request closed without merging**